lib/scope.js

```diff
@@ -838,8 +841,10 @@ module.exports = {
   // TODO: Change name to 'getMatchingRows'
   getMatchingRow: async function ( scope, { story_table, page_data }) {
     /* Given html_data and a list of objects describing fields and values,
-    * returns the matching fields and the data necessary to set those values.
-    * May also handle proxy vars (e.g. x, i...)
+    * returns a list of the matching fields and the data necessary to set
+    * those values: `[{ field: {}, table: {} }, ...]`. See
+    * test/unit_tests/matches.fixtures.js
+    * May in future also handle proxy vars (e.g. x, i...)
     * 
     * @param story_table {arr} - fields and the values they can be set to
     * @param story_table[n].var_name {str} - name of variable to set
```

Comment:
I think the `var_name` in this comment needs to be updated to `var`. 

Also, just out of curiosity, what is the reasoning behind renaming `var_name` to `var`?

Reply:
`var_name` was the renaming - it started as `var`. I just forgot what it was when I was writing the new code.